environment/spin-kube-k3s.sh

```diff
@@ -29,6 +29,10 @@ curl -sfL https://get.k3s.io | sh -s - server --write-kubeconfig-mode '0644' ${c
 
 export KUBECONFIG=/etc/rancher/k3s/k3s.yaml
 
+# Apply node labels used by install_* helpers from spin-kube.sh
+kubectl annotate node --all workload=system
```

Comment:
I am a little confused as to why we are annotating all nodes as system. Don't we just want to annotate the one system node?

Reply:
I see. Since that is one node for everything, i see how we need it to have the annotation